tests/common/test_cgroupstelemetry.py

```diff
@@ -392,6 +438,51 @@ def test_cgroup_is_tracked(self):
         self.assertFalse(CGroupsTelemetry.is_tracked("not_present_dummy_extension_{0}".format(i), 'cpu'))
         self.assertFalse(CGroupsTelemetry.is_tracked("not_present_dummy_extension_{0}".format(i), 'memory'))
 
+    @skip_if_predicate_false(CGroupConfigurator.get_instance().enabled, "CGroups not supported in this environment")
```

Comment:
Let's expose CGroupConfigurator._cgroups_supported as a public method and do this kind of checks based on that method